bindings/rascal/models/kernels.py

```diff
@@ -24,47 +32,94 @@ class Kernel(object):
         is between atomic environments) or 'Structure' (the kernel is summed over
         atoms in a structure), which is the default
 
+    kernel_type : string
+        Type of kernel method, either 'Full' (computing exact covariance matrix)
+        or 'Sparse' (computing GAP like kernel for sparse kernel methods like
+        Subset of Regressors)
```

Comment:
cite Quiñonero-Candela and Rasmussen if you're going to invoke Gaussian-process-specific jargon